tests/DotNetLightning.Core.Tests/Serialization.fs

```diff
@@ -742,8 +742,20 @@ module SerializationTest =
                 ()
                 
             testProperty "BitArray serialization" <| fun (ba : NonNull<byte[]>) ->
-                Expect.sequenceEqual (BitArray.FromBytes(ba.Get).ToByteArray()) (ba.Get) ""
-                
+                let backAndForth = BitArray.FromBytes(ba.Get).ToByteArray()
+                let finalArray = Array.zeroCreate ba.Get.Length
+                Array.Copy(backAndForth, 0, finalArray, ba.Get.Length - backAndForth.Length, backAndForth.Length)
+                Expect.isTrue false "tits"

```

Comment:
WTF

Reply:
Oh gosh some tests are not running  on CI.

Reply:
I made a PR to fix this #93 .
I want you to rebase this branch on master after merging #93 . @canndrew

Reply:
Oops :grimacing:

That was some dummy code to check that the test was actually running. I forgot to take it out.